CONTRIBUTING.md

```diff
@@ -1,10 +1,14 @@
-# Contributing Guidelines
+# Contributing to the Performance Lab plugin
 
-Welcome to the Performance plugin project! We're glad you're here. Here's a few things to help you get started.
+Thank you for your interest in contributing to the Performance Lab plugin! Complete documentation for getting started and contributing to the plugin is available in the [Performance Lab](https://make.wordpress.org/performance/handbook//performance-lab/) section of our [Performance Team Handbook](https://make.wordpress.org/performance/handbook/).
```

Comment:
```suggestion
Thank you for your interest in contributing to the Performance Lab plugin! Complete documentation for getting started and contributing to the plugin is available in the [Performance Lab](https://make.wordpress.org/performance/handbook/performance-lab/) section of our [Performance Team Handbook](https://make.wordpress.org/performance/handbook/).
```

Reply:
@mukeshpanchal27 Good catch! Should be fixed now.